core/src/main/scala/org/apache/spark/h2o/H2OContext.scala

```diff
@@ -255,7 +265,142 @@ object H2OContext extends Logging{
     * @return H2O Context
     */
   def getOrCreate(sc: SparkContext): H2OContext = {
-    getOrCreate(sc, new H2OConf(sc))
+    getOrCreate(sc, new H2OConf(sc))(SQLContext.getOrCreate(sc))
+  }
+
+  val defaultFieldNames = (i: Int) => "f" + i
```

Comment:
Not sure if I understood you correctly. You mean, moving H2OContext class
to converts?

Thanks,
-Vlad

On Thu, Aug 25, 2016 at 12:24 AM, Jakub Háva <notifications@github.com>
wrote:

> In core/src/main/scala/org/apache/spark/h2o/H2OContext.scala
> <https://github.com/h2oai/sparkling-water/pull/58#discussion_r76192534>:
>
> > @@ -255,7 +265,142 @@ object H2OContext extends Logging{
> >      * @return H2O Context
> >      */
> >    def getOrCreate(sc: SparkContext): H2OContext = {
> > -    getOrCreate(sc, new H2OConf(sc))
> > +    getOrCreate(sc, new H2OConf(sc))(SQLContext.getOrCreate(sc))
> > +  }
> > +
> > +  val defaultFieldNames = (i: Int) => "f" + i
>
> The code from here down should be located in org.apache.spark.converts
> package in order to keep h2o context clean. There are classes which takes
> care of converting various types such as RDD containing primitive type, RDD
> containing product class, dataframe to h2o frames and back.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/h2oai/sparkling-water/pull/58/files/dda8bacb29caf9d9ad724922e0bdf29e831b7ecb#r76192534>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAOuK0yTKckdjw0WAO11OdNtqVEfBfUyks5qjUMTgaJpZM4Jsgi_>
> .
>